flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/FlinkOperatorConfiguration.java

```diff
@@ -59,10 +59,11 @@ public class FlinkOperatorConfiguration {
     Duration flinkCancelJobTimeout;
     Duration flinkShutdownClusterTimeout;
     String artifactsBaseDir;
+    boolean savepointCleanerEnabled;
```

Comment:
I think this should not be part of the operator configuration (which are system wide configs) but instead should be configurable on a per resource level (simply get it based on the observeconfig)

Reply:
Move to ObserverConfig